doc/tech/approach.md

```diff
@@ -80,27 +80,19 @@ To address the concern areas detailed above the specific topics listed below wil
 
-•	**Abstraction**: A standardized layer between Virtualisation Layer domains and the Networking Hardware Layer resource pool hiding the implementation details of the Networking Hardware.
+-	**Abstraction**: A standardized layer between Virtualisation Layer domains and the Networking Hardware Layer resource pool hiding the implementation details of the Networking Hardware.
 > **Note:** Abstraction may be deployed in many different ways e.g. depending on target use case requirements, workload characteristics, different algorithm implementions of pipeline stages and available platforms. This includes Network Functions running on server nodes with or without programmable hardware acceleration, or networking support functions running on programmable network switches in the network.
-
-•	**Agnosticity** : Define Network Fabric concepts and models that can carry any type of traffic in terms of:
--	Control, User and Management traffic types
--	Acceleration technologies supporting multiple types of Infrastructure deployments and Network Function workloads 
-
-•	**Automation**: Enable full automation, from Physical Fabric installation and provisioning to automation of VNF/CNF onboarding.
-
-•	**Openness**: All Networking is based on open source or standardized APIs (NBI and SBI) and should enable integration of Open Source networking components (e.g. SDN controllers).
-
-•	**Programmability**: Should enable a programmable forwarding plane controlled from a separately deployed control plane that abstracts into standardized functions and interfaces.
-
-•	**Scalability**: Should be scalable to handle all traffic traverse North-South and East-West enabling small up to large deployments.
-
-•	**Workload diversity**: Should be capable to p=to provide connectivity to any type of workloads, including VNF , CNF and BareMetal workloads.
-
-•	**Telco Grade**: Should support deployments with Telco Grade characteristics requirements.
-
-•	**Future proof**: Should support known and emerging technology trends e.g. SmartNICs, FPGAs and Programmable Switches, multi-cloud aspects (public/private/hybrid), multi-location aspects (multi-cluster, DCI, etc), Edge realted technologies, etc.
-
+- **Agnosticity** : Define Network Fabric concepts and models that can carry any type of traffic in terms of:
+  - Control, User and Management traffic types
+  - Acceleration technologies supporting multiple types of Infrastructure deployments and Network Function workloads 
+-	**Automation**: Enable full automation, from Physical Fabric installation and provisioning to automation of VNF/CNF onboarding.
+-	**Openness**: All Networking is based on open source or standardized APIs (NBI and SBI) and should enable integration of Open Source networking components (e.g. SDN controllers).
+-	**Programmability**: Should enable a programmable forwarding plane controlled from a separately deployed control plane that abstracts into standardized functions and interfaces.
```

Comment:
Under Agnosticity, control, user and management planes were specified and so for consistency:
```suggestion
-	**Programmability**: Should enable a programmableuser (forwarding) plane controlled from a separately deployed control plane utilising open-source/standardized functions and interfaces.
```

Reply:
I agree with the user plane suggestion, but I think the utilisation of open source and/or standard interfaces is not really clear.

Reply:
@CsatariGergely the original wording included "standardized functions" -- I tried to generalise. I think the sentence is attempting to say that the "control" of the user plane utilises opensource and/or standadised functions and interfaces.

Reply:
Yes but this wording mixes the implementation using open source and the abstraction using standardized interfaces.